docs/reporting-services/tools/server-properties-advanced-page-reporting-services.md

```diff
@@ -97,9 +97,11 @@ A comma-separated list of headers that the server will expose to clients. The de
 **AccessControlMaxAge**
 Specifies the number of seconds the results of the preflight request can be cached. The default value is 600 (10 minutes).
 
-**AllowedResourceExtensionsForUpload** ***(Power BI Report Server only)***
-Set extensions of resources that can be uploaded to the report server. Extensions for built-in file types like &ast;.rdl and &ast;.pbix are not required to be included. Default is “&ast;, &ast;.xml, &ast;.xsd, &ast;.xsl, &ast;.png, &ast;.gif, &ast;.jpg, &ast;.tif, &ast;.jpeg, &ast;.tiff, &ast;.bmp, &ast;.pdf, &ast;.svg, &ast;.rtf, &ast;.txt, &ast;.doc, &ast;.docx, &ast;.pps, &ast;.ppt, &ast;.pptx”. 
+**AllowedResourceExtensionsForUpload**
+Set of extensions of resources that can be uploaded to the report server. Extensions for built-in file types like &ast;.rdl and &ast;.pbix are not required to be included. Default is “&ast;, &ast;.xml, &ast;.xsd, &ast;.xsl, &ast;.png, &ast;.gif, &ast;.jpg, &ast;.tif, &ast;.jpeg, &ast;.tiff, &ast;.bmp, &ast;.pdf, &ast;.svg, &ast;.rtf, &ast;.txt, &ast;.doc, &ast;.docx, &ast;.pps, &ast;.ppt, &ast;.pptx”.
 
+**RestrictedResourceMimeTypeForUpload**
+Set of mime types users are not allowed to upload content with. Any resources that are already stored with a restricted mime type will only be allowed to be downloaded as an application/octet-stream instead of being opened/executed by the browser.  By default, there are no restricted items in this list, but it is recommended organizations populate this to provide the most secure experience.

```

Comment:
```suggestion
Set of mime types users aren't allowed to upload content with. Any resources that are already stored with a restricted mime type can only be downloaded as an application/octet-stream instead of being opened/executed by the browser.  By default, there are no restricted items in this list, but we recommend that organizations populate this to provide the most secure experience.
```

Reply:
Sounds good

#signoff